docs/source/status.rst

```diff
@@ -44,6 +44,10 @@ Data Types
 +-------------------+-------+-------+-------+----+-------+-------+-------+-------+-----------+
 | Float32/64        | ✓     | ✓     | ✓     | ✓  |  ✓    |  ✓    | ✓     | ✓     | ✓         |
 +-------------------+-------+-------+-------+----+-------+-------+-------+-------+-----------+
+| Decimal32         | ✓     |       | ✓     |    |       |       |       |       |           |
++-------------------+-------+-------+-------+----+-------+-------+-------+-------+-----------+
+| Decimal64         | ✓     |       | ✓     |    |       |       |       |       |           |
++-------------------+-------+-------+-------+----+-------+-------+-------+-------+-----------+
```

Comment:
Go isn't part of this PR, is it?

Reply:
the Go impl was already merged in the arrow-go repo